File: reg.php
<?php

$API_KEY = "";

function device_reg() {
    global $API_KEY;
    if ($API_KEY == "") {
        throw new Exception("API key is required");
    }

    $headers = array(
        'x-api-key' => $API_KEY,
        'Content-Type' => 'application/json'
    );
    
    $curl = curl_init();
    curl_setopt_array($curl, array(
        CURLOPT_URL => "https://tiktokapi.dev/device-reg",
        CURLOPT_RETURNTRANSFER => true,
        CURLOPT_HTTPHEADER => array_map(function($key, $value) {
            return $key . ': ' . $value;
        }, array_keys($headers), $headers)
    ));
    $response = curl_exec($curl);
    curl_close($curl);
    return json_decode($response, true);
}

print_r(device_reg());

?>

File: reg.py
import requests
import json

API_KEY = "" # contact t.me/scrapguru for test key

def device_reg():
    if API_KEY == "":
        raise Exception("API key is required")

    headers = {
        'x-api-key': API_KEY,
        'Content-Type': 'application/json'
    }
    
    r = requests.get("https://tiktokapi.dev/device-reg", headers=headers)
    return r.json()

print(device_reg())
